server/src/main/java/org/elasticsearch/index/mapper/MappingLookup.java

```diff
@@ -137,6 +140,10 @@ public MappingLookup(Mapping mapping,
                 throw new MapperParsingException("Field [" + mapper.name() + "] is defined more than once");
             }
             indexAnalyzersMap.putAll(mapper.indexAnalyzers());
+            CheckedConsumer<IndexTimeScriptContext, IOException> postParsePhase = mapper.getPostParsePhase();
```

Comment:
I am thinking that clarifying and consolidating the notion of post parse phase, or whatever we want to call it, we could even make it more specific to script execution because that is all we need it for, would help making this more readable too.